keyboards/dygma/raise/create-led-config.js

```diff
@@ -0,0 +1,77 @@
+/* Copyright (C) 2021 Islam Sharabash
```

Comment:
Not sure this file belongs in the repo

Reply:
I used it to create the positions for the leds that are in the keymaps, I could remove it, but I think it's nicer for anyone in the future to be able to see where those numbers came from. Let me know if you want it removed.